src/introspection.js

```diff
@@ -0,0 +1,47 @@
+const fetch = require('node-fetch');
+const id = process.env.client_id;
+const secret = process.env.client_secret;
+const openidURL = process.env.OPEN_ID_ENDPOINT;
+

```

Comment:
We could move the env settings to ./src/config.js to have a differentiation between dev / test / prod.